src/history.js

```diff
@@ -40,7 +41,9 @@ function CommitInfo({ commit, move, onClick }) {
             {commit.author.login}
           </div>
           <div style={{ fontSize: "0.85rem", opacity: "0.9" }}>
-            on {commit.date.toDateString()}
+            <a href={Activelink} target="_blank">
+              <div>{commit.date.toDateString()}</div>
+            </a>
           </div>
```

Comment:
You can do something like this inside the div:
```jsx
  {isActive && commit.commitUrl ? (
    <a href={commit.commitUrl} target="_blank">
      on {commit.date.toDateString()}
    </a>
  ) : (
    `on ${commit.date.toDateString()}`
  )}
```

Reply:
Ah I was trying to do it inside the div, but using `if`. that's why it was failing me.